v21.1/show-backup.md

```diff
@@ -301,6 +302,87 @@ Or, use the `kms` option and the same KMS URI that was used to create the backup
 (20 rows)
 ~~~
 
+### Show a backup's manifest
+
+Use the `WITH as_json` option to output a backup's manifest as a JSON value:
```

Comment:
ditto above

No matter what we call it, we should explicitly state that this is for inspecting the _internal_ metadata, for example for debugging, and is not a stable public API, and its contents may change in the future, and that users should prefer to use a different `SHOW BACKUP` form if they can, or if they can't, file an issue describing the use-case that is not supported by the other (stable) SHOW BACKUP APIs

Reply:
Great points @dt, I've revised and drafted with "manifest" de-emphasized. I have retained it in places — though always clarifying what it contains, so it does have meaning to the user. My reasoning is that since the `as_json` option displays within a `manifest` column, referring to the metadata as within the manifest provides some context for the user on that front at least. Let me know what you think of these changes. 

I've also added a draft note to describe the usage as debugging. I'm wondering if this needs to be stronger, maybe a warning? @livlobo I assume we don't necessarily need to go as strong as only suggesting usage of this with the Support team.

Reply:
It is certainly _safe_ to use this without any sort of support guidance / supervision, since it is just reading, but it just is not something we're making any promises about what it'll report, and is not subject to the same external API stability guarantees as the rest of cockroach's SQL surface area including the other flavors of `SHOW BACKUP`. Its content *will* change from version to version, and should not be used for anything other than ad-hoc manual inspection.

Reply:
Thanks for the confirmation. OK, with that in mind, it seems me the note in place would suffice to provide enough recommendation to only use this for debugging or inspection. Other than that I have directed toward using other `SHOW BACKUP` options for more stable responses. Are you OK with the other changes @dt?